App/constants/strings.js

```diff
@@ -1,6 +1,10 @@
 const Strings = {
   SelectedOrganization: 'SelectedOrganization',
   RecentlyViewed: 'id',
+  CurrentSelectedMediaId: 'CurrentSelectedMediaId',
+  Address: 'address',
+  FindMediaCenters: 'FindMediaCenters',
+  Square: 'square',
```

Comment:
If a string is being used just once you can declare it in that file. like => const SQUARE = 'square' and use.